src/GraphRenderer.ts

```diff
@@ -794,9 +794,10 @@ export class GraphRenderer {
 
         //determining if the edge's length is below a certain threshold. if it is we will draw the edge straight and remove the arrow
         const isShortEdge: boolean = edgeLength < EagleConfig.STRAIGHT_EDGE_SWITCH_DISTANCE;
+        const hideArrow: boolean = edgeLength < EagleConfig.EDGE_DISTANCE_ARROW_VISIBILITY;
 
         if (edge !== null){
-            edge.setIsShortEdge(isShortEdge)
+            edge.setIsShortEdge(hideArrow)
```

Comment:
I guess we should probably change the name of the setIsShortEdge() function here.

If you are calculating whether to hide the arrow, and storing that value on the edge, then "isShortEdge" is not a good name. Change it to 'hideArrow' or something similar.

Or do we even need to store the value of hideArrow on the edge? Perhaps we can remove the "isShortEdge" variable in the Edge class (and the get/set functions? It looks like we only use it locally within this function?

Reply:
i was able to remove it and handle arrow visibility within the edge drawing function without the need for a ko data bind